MainModule/Server/Commands/HeadAdmins.lua

```diff
@@ -100,11 +100,11 @@ return function(Vargs, env)
 			end
 		};
 
-		GameBan = {
+		PermenantBan = {
 			Prefix = Settings.Prefix;
-			Commands = {"gameban", "saveban", "databan", "pban"};
+			Commands = {"permenantban", "permban", "pban"};
```

Comment:
```suggestion
			Commands = {"permenantban", "permban", "pban"};
			Commands = {"permenantban", "permban", "pban", "gameban", "saveban", "databan"};
```
This will prevent people from being confused if they use an alias such as `:gameban`. If removed, they may get the impression that they can no longer game ban people